pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1035,13 +1203,38 @@ def _eigvals(cls, *params):
         return qml.math.stack([1, 1, exp_part, qml.math.conj(exp_part)])
 
     @staticmethod
-    def decomposition(lam, wires):
-        decomp_ops = [
-            PhaseShift(lam / 2, wires=wires[1]),
+    def compute_decomposition(phi, wires):
+        """Determine ``CRZ``'s decomposition for specified wire.
+        The decomposition defines an Operator as a product of more fundamental gates:
+
+        .. math:: O = O_1 O_2 \dots O_n.
+
+        ``compute_decomposition`` is a static method and can provide the decomposition of a given
+        operator without creating a specific instance. The instance method ``decomposition`` uses
+        this method and the instance's variables.
+
+        Args:
+            phi (float): rotation angle :math:`\phi`
+            wires (Any, Wires): Wires that the operator acts on.
+
+        Returns:
+            tuple[Operator]: decomposition into lower level operations
+
+        **Example:**
```

Comment:
Just realised, Josh always corrects my funky versions of the Example header, and I think we don't use the colon...but happy to overlook it here :) We can change it globally by find and replace one day.